src/plugin.js

```diff
@@ -231,18 +231,45 @@ const onPlayerReady = (player) => {
  * to you; if not, remove the wait for "ready"!
  *
  * @function eme
+ * @param    {Object} [player]
+ *           A playerobject.
  * @param    {Object} [options={}]
  *           An object of options left to the plugin author to define.
  */
-const eme = function(options = {}) {
-  this.eme.options = options;
+const eme = function(player, options = {}) {
+  const initializeMediaKeys = (emeOptions = {}) => {
+    const e = {
```

Comment:
I think we should use a more descriptive variable name. Maybe something like `mockEncryptedEvent` or something alongside a comment.